rm-datasource/src/main/java/io/seata/rm/datasource/exec/BaseTransactionalExecutor.java

```diff
@@ -542,18 +559,15 @@ protected List<String> getNeedColumns(String table, String tableAlias, List<Stri
                         .collect(Collectors.toList());
             }
             onUpdateColumns.removeAll(unescapeColumns);
-            needUpdateColumns.addAll(onUpdateColumns.stream()
-                .map(onUpdateColumn -> ColumnUtils.addEscape(onUpdateColumn, getDbType(), tableMeta))
-                .collect(Collectors.toList()));
+            needUpdateColumns.addAll(onUpdateColumns);
         } else {
             Stream<String> allColumns = tableMeta.getAllColumns().keySet().stream();
             if (StringUtils.isNotBlank(tableAlias)) {
                 allColumns = allColumns.map(columnName -> getColumnNameWithTablePrefix(table, tableAlias, columnName));
             }
-            allColumns = allColumns.map(columnName -> ColumnUtils.addEscape(columnName, getDbType(), tableMeta));
             allColumns.forEach(needUpdateColumns::add);
         }
-        return needUpdateColumns;
+        return needUpdateColumns.parallelStream().map(column -> ColumnUtils.addEscape(column, getDbType(), tableMeta)).collect(Collectors.toList());
```

Comment:
```suggestion
        return needUpdateColumns.stream().map(column -> ColumnUtils.addEscape(column, getDbType(), tableMeta)).collect(Collectors.toList());
```
With stream, when there are not many columns involved, stream is theoretically better.